docs/intro/1_whats_in_the_box.md

```diff
@@ -48,6 +51,7 @@ and provide the following **technology stack**:
 - **Cordova** and it's **CLI** for accessing phone APIs and building the app - http://cordova.apache.org/
   - **ngCordova** connecting Angular with Cordova http://ngcordova.com/
 - **SASS** as the CSS preprocessor of choice - http://sass-lang.com/
+- **JADE** as the HTML template engine of choice - http://jade-lang.com/
```

Comment:
I do not agree here, Jade is super powerful and very easy, this should be recommended as a best practice over classical html. It's like using css instead of sass.